nativecfg/reflect-config-cleaned.json

```diff
@@ -0,0 +1,20 @@
+[ {
+  "name" : "java.io.File",
+  "allPublicMethods" : true
+}, {
+  "name" : "java.lang.Object[]"
+}, {
+  "name" : "java.lang.String",
+  "allPublicMethods" : true
+}, {
+  "name" : "java.lang.String[]"
+}, {
+  "name" : "java.lang.reflect.AccessibleObject",
+  "methods" : [ {
+    "name" : "canAccess",
+    "parameterTypes" : [ "java.lang.Object" ]
+  } ]
+}, {
+  "name" : "java.util.Properties",
```

Comment:
This really should not end up in the reflect config. If it does, then I consider that a bug in refl.

See https://clojure.atlassian.net/browse/CLJ-2636